dcplib/etl/__init__.py

```diff
@@ -60,33 +65,50 @@ def dss_client(self):
             self._dss_client = hca.dss.DSSClient(swagger_url=self._dss_swagger_url)
         return self._dss_client
 
-    def link_file(self, bundle_uuid, bundle_version, f):
-        if not os.path.exists(f"{self.sd}/bundles/{bundle_uuid}.{bundle_version}/{f['name']}"):
-            logger.debug("Linking fetched file %s/%s", bundle_uuid, f["name"])
-            os.symlink(f"../../files/{f['uuid']}.{f['version']}",
-                       f"{self.sd}/bundles/{bundle_uuid}.{bundle_version}/{f['name']}")
+    def etl_one_bundle(self, bundle_uuid, bundle_version):
```

Comment:
extract felt like it didnt quite encompass what the function was actually doing (as its also applying the transform and load functions) but I think it makes sense to leave it to avoid breaking current usages of the extractor